docs/source/maintenance/add_peer.rst

```diff
@@ -0,0 +1,61 @@
+============
+Adding Peers
+============
+
+In HL Iroha, you can add new peers to the network while it is running.
+This is done by using a special command, `AddPeer <../develop/api/commands.html#add-peer>`_.
+
+Requirements
+============
+
+**There should be a peer that:**
+
+— runs with a Genesis Block (initial block of the blockchain) identical to the one on the peers already in the network;
+
+— has a resolvable address;
+
+— has a peer keypair (Ed25519 with SHA-3)
+
+**The account that is sending the transaction adding a peer must have a `root permission <../develop/api/permissions.html#root>`_ in their role - this must be set in the genesis block.**
```

Comment:
why isn't `can_add_peer` enough?

Reply:
I was told that root is needed somehow. @lebdron , maybe you know why it could be needed?

Reply:
Fixed!